draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -276,55 +274,57 @@
-				<t>Type: 1.</t>
-				<t>Length: 4 octets.</t>
-				<t>Value: number of un-acknowledged LSPs which can be sent back to back.</t>
-				<t>Note that if an LSP has not been acknowledged and is sent again, it does not count twice. The reason is that this LSP is assumed to be lost and hence not in a buffer at the receiver.</t>
+			<t>For a given IS-IS adjacency, the Flooding Parameters TLV does not need to be advertised in each IIH or PSNP. The LSP transmitter uses the latest received value for each parameter until a new value is advertised by the LSP receiver. However, IIHs and PSNPs are not reliably exchanged, and may never be received, so parameters MAY be sent even if there is no change in value since the last transmission. For a parameter which has never been advertised, the LSP transmitter SHOULD use its local default value. That value SHOULD be configurable on a per node basis and MAY be configurable on a per interface basis.</t>
+			<section anchor="LSPReceptionWindow" title="LSP Burst Window sub-TLV">
+				<t>The LSP Burst Window sub-TLV advertises the maximum number of un-acknowledged LSPs that the node can receive with no separation interval between LSPs.</t>
+				<t>Type: 1</t>
+				<t>Length: 4 octets</t>
+				<t>Value: number of un-acknowledged LSPs that can be sent back to back</t>
+				<t>Note that if an LSP has not been acknowledged and is sent again, it does not count twice. The reason is that this LSP is assumed to be lost and hence not in a buffer at the receiver.
+				[This requires some clarification. Why would we send the same LSP twice in the same burst? If it is a retransmission of an LSP that was originally sent outside of the current burst, then shouldn't it count as part of the burst? -- Tony L.]</t>
+
 			</section>
-			<section anchor="InterfaceLSPTransmissionInterval" title="minimumInterfaceLSPTransmissionInterval sub-TLV">
-				<t>The sub-TLV minimumInterfaceLSPTransmissionInterval advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be processed/received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
-				<t>Type: 2.</t>
-				<t>Length: 4 octets.</t>
-				<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the receive window has been used.</t>
+			<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
+				<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
+				<t>Type: 2</t>
+				<t>Length: 4 octets</t>
+				<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the burst window has been used</t>
```

Comment:
ISO10589 controls LSPs/second. So you can burst - but the total # of
LSPs/second cannot exceed the defined rate. That would seem to apply here -
otherwise a strict interpretation does not limit how often a burst can be sent.
Do you agree? Then we need to reword.

Reply:
@Les that's not how this is designed.
- You can send a Burst every RTT (as per the usual receive window algo)
- But to account for very high RTT or all LSP been lost hence available window been nullified, we need a minimal allowed rate. "LSP Transmission Interval sub-TLV" is this minimal allowed rate.

Reply:
I don't think you mean that you want to burst every RTT.  I think we want to say that you can burst if there is capacity on the receiver side.  See my proposed text in the previous comment.